components/core/src/clp/string_utils/string_utils.hpp

```diff
@@ -126,19 +126,11 @@ bool convert_string_to_int(std::string_view raw, integer_t& converted);
 
 template <typename integer_t>
 bool convert_string_to_int(std::string_view raw, integer_t& converted) {
-#if defined(_MSC_VER)
-    auto* raw_begin = raw.data();
-    auto* raw_end = raw_begin + raw.size();
-#else
-    auto raw_begin = raw.cbegin();
-    auto raw_end = raw.cend();
-#endif
-    auto result = std::from_chars(raw_begin, raw_end, converted);
-    if (raw_end != result.ptr) {
-        return false;
-    } else {
-        return result.ec == std::errc();
-    }
+    auto* raw_begin{raw.data()};
+    auto* raw_end{raw_begin + raw.size()};
```

Comment:
```suggestion
    auto const* raw_begin{raw.data()};
    auto const* raw_end{raw_begin + raw.size()};
```
Althrough the type deduction will result a const ptr, but adding a const explicitly can make it less misleading